mobile/src/libs/components/button/button.tsx

```diff
@@ -1,43 +1,56 @@
 import React from 'react';
 
-import { Pressable, Text } from '#libs/components/components';
+import { Icon, Pressable, Text } from '#libs/components/components';
+import { AppColor } from '#libs/enums/enums';
+import { type IconName } from '#libs/types/types';
 
 import { styles } from './styles';
 
 type Properties = {
   label?: string;
   onPress: () => void;
   isDisabled?: boolean;
-  type?: 'solid' | 'outlined';
+  type?: 'solid' | 'outlined' | 'rounded' | 'transparent';
```

Comment:
Let's separate 'rounded' from others. Rounded is more about the shape of the button, but others are more about the color